audit/audit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x -e
+CNCF_GCP_ORG=758905017065
+
+# gcloud organizations describe $CNCF_GCP_ORG 2>&1
+# ERROR: (gcloud.organizations.describe)
+# User [hh@ii.coop] does not have permission to access organization []
+
+for format in json yaml
+do
+    gcloud organizations get-iam-policy $CNCF_GCP_ORG --format=$format \
+           > cncf-org-policy.$format
```

Comment:
nit: can we use file names with periods as delimiters, since project names can't include periods?  E.g. `cncf-org.policy.$format`  so `*.policy.*` is all policy lists and `*.roles.*` is all roles lists